capacitron_inference.py

```diff
@@ -0,0 +1,64 @@
+'''
+This will be deleted later, only for dev, to see how to infer the capacitron model
```

Comment:
This file will be deleted before the merge, it's only a script to show how to infer the model for others who are experimenting with this work